.readthedocs.yaml

```diff
@@ -0,0 +1,19 @@
+# Configuration file for ReadTheDocs, used to render the CV32E40X
```

Comment:
This file should not get merged. There is a separate PR for that already and I first want to see if there are no issues with the 40X user manual.